SolidStateGroup/bullet-train-api
# chore: Dynamically add/remove albacross

Thanks for submitting a PR! Please check the boxes below:


## Changes

- Dynamically add/remove Albacross script on the Signup page

## How did you test this code?
- See Developer Tools > Network, filter by the word "albacross"
- Go to Signup page
